src/Notifications.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+
+interface NotificationsProps {
+  removeTodoIsClicked: boolean,
+  setRemoveTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+  editTodoIsClicked: boolean,
+  setEditTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+  onEmptyFormSubmit: boolean,
+  setOnEmptyFormSubmit: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const Notifications: React.FC<NotificationsProps> = ({
+  removeTodoIsClicked,
+  setRemoveTodoIsClicked,
+  editTodoIsClicked,
+  setEditTodoIsClicked,
+  onEmptyFormSubmit,
+  setOnEmptyFormSubmit,
+}) => {
+  const [isNotificationHidden,
+    setIsNotificationHidden] = useState(false);
+
+  const hideNotification = () => {
+    setIsNotificationHidden(true);
+    setRemoveTodoIsClicked(false);
+    setEditTodoIsClicked(false);
+    setOnEmptyFormSubmit(false);
+  };
+
+  const showErrorText = () => {
```

Comment:
The function 'showErrorText' should return a default error message in case none of the conditions match. This is to ensure that an error message is always shown when the notification is visible.